package-lock.json

```diff
@@ -44,6 +44,12 @@
                         "ms": "^2.1.1"
```

Comment:
One last thing, the package-lock change should be reverted. Thanks for the contribution!